### PR TITLE
Feat: show contributors & fix mulptiple page reload after F5

### PR DESCRIPTION
**Pull request by ursarik**

### CODE REVIEW

File: backend/src/data/seed-data/page-content.data.ts [hunk 1]
```diff
@@ -99,12 +99,21 @@ export const pageContents = [
...
     authorId: '3541af71-9d5b-4ca5-a74a-f629aea76735',
   },
+  {
+    id: 'd7fe5fb3-58a9-40d5-b2f0-11be41b0880e',
+    pageId: 'bc4ee38b-56dc-4202-a1e2-48ae5e7affda',
+    createdAt: new Date(),
+    title: 'Bitwise AND (&)',
+    content: 'The bitwise AND operator (&) returns a 1 in each bit position for which the corresponding bits of both operands are 1s. Syntax: a & b',
+    authorId: 'bf873638-9388-4298-94b0-3d1a11ef8688',
+  },
```
Comment (mykytabatrak) [inline backend/src/data/seed-data/page-content.data.ts]: You don't need to provide id and createdAd properties, they are supplied by database as default values

File: backend/src/data/seed-data/user.data.ts [hunk 2]
```diff
@@ -22,4 +22,11 @@ export const users = [
     avatar: '',
     password,
   },
+  {
+    id: 'bf873638-9388-4298-94b0-3d1a11ef8688',
```
Comment (mykytabatrak) [inline backend/src/data/seed-data/user.data.ts:26]: Again, id is unnecessary

File: frontend/src/components/pages/components/page-contributors/page-contributors.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { Card, Badge } from 'react-bootstrap';
+import Avatar from 'react-avatar';
+import { IPageContributor } from 'common/interfaces/page';
+import { useHistory } from 'react-router-dom';
+import { AppRoute } from '../../../../common/enums/enums';
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/pages/components/page-contributors/page-contributors.tsx]:
```suggestion
import { AppRoute } from 'common/enums/enums';
```

File: frontend/src/components/pages/components/page-contributors/page-contributors.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  const avatarStyles: React.CSSProperties = {
+    boxSizing: 'content-box',
+    border: '4px solid white',
+    marginLeft: -(+avatarSize / 2.2) + 'px',
+    cursor: 'pointer',
+  };
+
+  const handleAvatarClick = (userId: string): void => {
+    history.push(
+      AppRoute.PROFILE.slice(0, AppRoute.PROFILE.length - 3) + userId,
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/pages/components/page-contributors/page-contributors.tsx]: I think it is time to create a helper for this. Please, search in code all places with `.length - 3` and replace them with new helper everywhere

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

File: backend/src/data/seed-data/page-content.data.ts (hunk 1 above)
Comment (aleksandr-kuharenko) [reply backend/src/data/seed-data/page-content.data.ts]: Id is necessary for seeders. We should have the same data in DB after running the seeds.

**Comment by aleksandr-kuharenko** [comment]
Please revert back IDs in seeders: https://github.com/BinaryStudioAcademy/bsa-2021-infostack/pull/90#discussion_r686342554

**Review by mykytabatrak — APPROVED** [review]

**Review by aleksandr-kuharenko — APPROVED** [review]

**Pull request merged**